api-tests/src/types/cri-stub.ts

```diff
@@ -34,10 +34,12 @@ interface CriStubUserInfoEndpointErrorRequest {
 
 export interface CriStubResponse {
   redirectUri: string;
-  jarPayload: {
-    context?: string;
-    evidence_requested?: { strength?: number; validity?: number };
-  };
+  jarPayload: CriStubResponseJarPayload;
+}
+
+export interface CriStubResponseJarPayload {
```

Comment:
(I wonder if it's worth adding a note that this is actually only a small subset of this interface - it's missing all the 'standard' properties!)